components/greaselion/browser/greaselion_converted_extension.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_GREASELION_BROWSER_GREASELION_CONVERTED_EXTENSION_H_
+#define BRAVE_COMPONENTS_GREASELION_BROWSER_GREASELION_CONVERTED_EXTENSION_H_
+
+#include <memory>
+#include <utility>
+
+#include "base/files/scoped_temp_dir.h"
+#include "base/macros.h"
+#include "base/memory/weak_ptr.h"
+
+namespace extensions {
+class Extension;
+}  // namespace extensions
+
+namespace greaselion {
+
+class GreaselionConvertedExtension {
```

Comment:
Not sure whether we need this class, can we just return `std::pair`? Or it could be base::Optional<std::pair<...>>` if you need to distinguish an "empty" case

Reply:
Yes, will do.